Client/Application/ImageIdentifiers.swift

```diff
@@ -32,6 +32,7 @@ public struct ImageIdentifiers {
     public static let key = "key"
     public static let libraryPanelHistory = "library-history"
     public static let libraryPanelSearch = "search"
+    public static let libraryPanelDelete = "action_delete"
```

Comment:
I believe we already have a `trashIcon`. Is this an image duplicate? 🤔 If it is, we should mark it in the TODO to address later and stick to one.

Reply:
I'm reusing one of the trashIcon we have in the app, I just added to the list of ImageIdentifiers